File: Atributos de validación Data Annotations multiidioma/[VB]-Atributos de validación Data Annotations multiidioma/VB/LocalizeDataAnnotations/Models/PersonaBBDD.vb
Imports System.ComponentModel.DataAnnotations

Public Class PersonaBBDD
    Implements IPersona

    <Required(ErrorMessageResourceName:="NombreObligatorio" _
        , ErrorMessageResourceType:=GetType(TextosLoader))> _
    <Display(Name:="Nombre", ResourceType:=GetType(TextosLoader))> _
    Public Property Nombre As String Implements IPersona.Nombre

    <Display(Name:="Apellido", ResourceType:=GetType(TextosLoader))> _
    Public Property Apellido As String Implements IPersona.Apellido

    <Display(Name:="Ciudad", ResourceType:=GetType(TextosLoader))> _
    <StringLength(20, MinimumLength:=4, ErrorMessageResourceName:="CiudadErrorLongitud" _
        , ErrorMessageResourceType:=GetType(TextosLoader))> _
    Public Property Ciudad As String Implements IPersona.Ciudad

    <Display(Name:="Email", ResourceType:=GetType(TextosLoader))> _
    Public Property Email As String Implements IPersona.Email

End Class

File: Atributos de validación Data Annotations multiidioma/[C#]-Atributos de validación Data Annotations multiidioma/C#/LocalizeDataAnnotations/Models/PersonaBBDD.cs
using LocalizeDataAnnotations.Infrastructure;
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Web;

namespace LocalizeDataAnnotations.Models
{
    public class PersonaBBDD: IPersona
    {
        [Required(ErrorMessageResourceName = "NombreObligatorio"
            , ErrorMessageResourceType = typeof(TextosLoader))]
        [Display(Name = "Nombre", ResourceType = typeof(TextosLoader))]
        public string Nombre { get; set; }

        [Display(Name = "Apellido", ResourceType = typeof(TextosLoader))]
        public string Apellido { get; set; }

        [Display(Name = "Ciudad", ResourceType = typeof(TextosLoader))]
        [StringLength(20, MinimumLength = 4, ErrorMessageResourceName = "CiudadErrorLongitud"
            , ErrorMessageResourceType = typeof(TextosLoader))]
        public string Ciudad { get; set; }

        [Display(Name = "Email", ResourceType = typeof(TextosLoader))]
        public string Email { get; set; }

    }
}